oak-store-spi/src/main/java/org/apache/jackrabbit/oak/plugins/memory/AbstractBlob.java

```diff
@@ -152,7 +122,7 @@ public String getContentIdentity() {
 
     /**
      * To {@code Blob} instances are considered equal iff they have the
-     * same SHA-256 hash code  are equal.
+     * same SHA-256 hash code or are equal.
```

Comment:
s/or are equal/or are identical/ would make clearer that we refer to ==, not equal() here.

Reply:
Sill totally redundant though.